### ISSUE TITLE
Several boolean attributes are not created as such

### ISSUE THREAD

**Issue by mithrandi** [meiersi/blaze-react#24]
I noticed `disabled` and `defer`, but there are no doubt more of these. I can't find the `src/Util/GenerateHtmlCombinators.hs` that the source claims is the generator of the code for the attributes, so I'm not sure what the correct way to fix these is.

**Comment by asayers** [comment]
Ah, those comments are no longer accurate, and should be removed. The `Text.Blaze.Html5.Attributes` module was originally copied from the `blaze-html` library for API compatibility, but we've diverged a bit and this module can no longer be generated by `GenerateHtmlCombinators.hs`.

If you're happy to fix those attributes, please do so directly in `Text.Blaze.Html5.Attributes`. Thanks!

**Comment by mithrandi** [comment]
Okay, good to know. I'm going to see if I can systematically extract all of the boolean attributes via the HTML5 spec, rather than playing whack-a-mole with individual attributes.

**Comment by asayers** [comment]
That sounds good!

**Comment by mithrandi** [comment]
Since we're using the React virtual DOM, the correct source of information is, in fact, here: https://github.com/facebook/react/blob/master/src/browser/ui/dom/HTMLDOMPropertyConfig.js#L47

I updated the attributes manually based on this information; I guess doing it programmatically would have been better, but I was too lazy. PR incoming as soon as I go over the diff again!

**Comment by asayers** [comment]
Yes, that looks correct, although the version of react.js we're using is 0.12.2. Relevant diff:

```diff
$ git diff v0.12.2..master -- src/browser/ui/dom/HTMLDOMPropertyConfig.js 
+    headers: null,
+    high: null,
+    low: null,
-    open: null,
+    open: HAS_BOOLEAN_VALUE,
+    optimum: null,
+    scoped: HAS_BOOLEAN_VALUE,
+    itemID: MUST_USE_ATTRIBUTE,
+    itemRef: MUST_USE_ATTRIBUTE,
```

I'd say "just bump the version", but `high`, `low`, `optimum` and `scoped` aren't in 0.13.1 either, which is the latest release. I suggest removing these attributes from the PR - keep them around for later though. Thoughts?

**Comment by mithrandi** [comment]
Ah, makes sense. It's not too hard to add them later, just need to remember about it when the React we're using is bumped; as far as I can tell, they won't actually work properly without being present in the React mapping, so no point having them around to confuse people.

I think I'll remove "open" entirely as well, for now, since it won't work correctly with the React property mapping having the wrong type for it.

Anyhow, PR updated (I didn't squash the commits since I tend to prefer preserving history, but let me know if you would prefer them squashed).

**Comment by asayers** [comment]
Fixed by bff603b19f38d92d9
